### PR TITLE
Documentation for kaiser commands

### PR DESCRIPTION
**Pull request by davidsiaw**
I tried to document all the commands that I was too busy to do so before. This will hopefully make the tool less mysterious.

You can review this here: https://tech.degica.com/kaiser/branches/documentation/0115-command-reference/

### CODE REVIEW

**Comment by degikko** [comment]
@showwin can you help us review this PR, please?

File: docs/pages/0115-command-reference.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+title: Command Reference
+
+---
+
+## Command Reference
```
Comment (showwin) [inline docs/pages/0115-command-reference.md:5]: The first 4 lines might be redundant?
```suggestion
## Command Reference
```
Screenshot 2024-05-08 at 16 49 20

**Review by showwin — APPROVED** [review]
Great! Thank you!

File: docs/pages/0115-command-reference.md (hunk 1 above)
Comment (davidsiaw) [reply docs/pages/0115-command-reference.md:5]: those are called the front matter for the html generator to use

Comment (showwin) [reply docs/pages/0115-command-reference.md:5]: Oh really, interesting 😮 
Thank you for letting me know.

**Pull request merged**